client/webpack.config.js

```diff
@@ -0,0 +1,64 @@
+const path = require('path');
+const webpack = require('webpack');
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+
+module.exports = {
+    entry: [
+        'webpack-dev-server/client?http://localhost:8000', // WebpackDevServer host and port
```

Comment:
For production bundle I suggest to use another configuration file, with source-maps, without hot-reload, etc.

Something like this:

```
    devtool: 'source-map',
    entry: {
        app: './app/index'
    },
    output: {
        path: path.join(__dirname, 'dist'),
        filename: 'bundle.js',
        publicPath: './'
    }
```